bridge_ui/package.json

```diff
@@ -23,7 +23,7 @@
     "@solana/web3.js": "^1.35.1",
     "@terra-money/wallet-provider": "^3.8.0",
     "@alephium/walletconnect-qrcode-modal": "0.1.0",
-    "@alephium/walletconnect-ethereum-provider": "0.1.1",
+    "@walletconnect/ethereum-provider": "2.17.2",
```

Comment:
In the latest WalletConnect, we will separate the WalletConnect storage for Alephium and Ethereum. So we no longer need to maintain a custom Ethereum provider.